CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ swagger 2 and OpenAPI 3.
 ### Defects Corrected
 * Fixed health DTOs Jackson deserialization.
 
+## 2.9 - September 5 2018
```

Comment:
Should this go with the 3.1 release? We could backport it to the 2.x line if needed.

Reply:
I had suggested using `@since 2.9` in the code since we would likely backport it, but I now realize the changelog complicates things a little bit.

Reply:
Yeah I was unsure what to do with the changelog. Is what I have here acceptable or do we need to do something else?

Reply:
@johnlcox and I talked. Lets have this enhancement go out on 3.1 release of Beadledom. Going from 2.x to 3.x is trivial, http://engineering.cerner.com/beadledom/3.0/docs/releases/Beadledom30.html